pyproject.toml

```diff
@@ -87,7 +87,7 @@ path = ".venv"
 test        = "pytest -n 4 --cov src --cov-report=xml --timeout 30 tests/unit --durations 20"
 coverage    = "pytest -n auto --cov src tests/unit --timeout 30 --cov-report=html --durations 20"
 integration = "pytest -n 10 --cov src tests/integration --durations 20"
-fmt         = ["black .",
+fmt         = ["black . --extend-exclude 'tests/unit/source_code/samples/*'",
```

Comment:
pylint excludes would also be necessary, but let's take the rename out of scope of this pr.

Reply:
done